### ISSUE TITLE
[A11Y] Navigation landmark missing label

### ISSUE THREAD

**Issue by israelcefrin** [pkp/immersion#126]
## Issue Description
There are three navigation landmark `<nav>`, but two of them are missing label or title.

## How to reproduce / Current behavior
Check the HTML structure viewing the source code and notice that the navigation landmarks don't present `aria-label` neither `aria-labelledby` attributes.

## What application are you using?
 OJS version 3.4

## Stack
- Device:  Laptop, 
- OS:  MacOS Sequoia 
- Browser:  edge, safari, chrome

## Testing method
- Method: Manual, WAVE plugin

## Action
Implement the aria attributes to relate the components with the heading tags with their labels.

## Additional information
Navigation components missing labels:

<img width="146" alt="Image" src="https://github.com/user-attachments/assets/2be3e431-4a24-4ead-8ad3-729b90c47d34" />

**Comment by israelcefrin** [comment]
Hi @kaitlinnewson 
Just dropping a note about the PR ready for review.
